integration_test/search_test.go

```diff
@@ -47,6 +47,18 @@ func TestKrewSearchOne(t *testing.T) {
 	}
 }
 
+func TestKrewSearchByNameAndDesc(t *testing.T) {
+	skipShort(t)
+
+	test := NewTest(t)
+
+	plugins := lines(test.WithDefaultIndex().Krew("search", "kubeconfig").RunOrFailOutput())
+	if len(plugins) < 11 {
+		// the first line is the header
+		t.Errorf("Expected at least %d plugins", len(plugins)-1)
+	}
```

Comment:
This is not a very good test, I think at this point we should extract the core search functionality into a separate func and unit-test that. The integration tests for this still could be useful but they should probably test something like "a particular plugin that doesn't have x in the name but x in the description".